### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by H-Bo214**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
* Added additional tests in Header
* Add test for logout, should display default Header state
* Moved getSinglePokemon to apiCalls.js
* Updated testData
* Imported getSinglePokemonData to App
### Files modified:
- [ ] index.html
- [x] App
- [ ] CardContainer
- [ ] PokeCard
- [ ] PokeForm
- [ ] Login
- [ ] FormContainer
- [ ] index.js
- [x] Other files: Header.test.js, App.test.js

### CODE REVIEW

File: stretch-auth/src/App/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import CardContainer from "../CardContainer/CardContainer";
 import FormContainer from "../FormContainer/FormContainer";
 import "./App.css";
 import { secrets } from "../secrets";
+import {getSinglePokemonData} from '../apiCalls'
```
Comment (AurumValian) [inline stretch-auth/src/App/App.js:9]: Good job moving this over!

**Review by AurumValian — APPROVED** [review]
Liking the tests so far!

File: stretch-auth/src/App/App.js (hunk 1 above)
Comment (jordy1611) [reply stretch-auth/src/App/App.js:9]: Good work! Hard to keep all that organized

**Review by jordy1611 — APPROVED** [review]
rad!

**Review by nware1066 — APPROVED** [review]
thanks for walking through this with me.

**Pull request merged**